fastapi_azure_auth/user.py

```diff
@@ -1,16 +1,247 @@
+        default=None,
+        description='Region of the resource tenant',
+    )
+    upn: Optional[str] = Field(
+        default=None,
+        description='An identifier for the user that can be used with the username_hint parameter; not a durable identifier for the user and should not be used to key data',
+    )
+    verified_primary_email: Optional[List[str]] = Field(
+        default=None,
+        description="Sourced from the user's PrimaryAuthoritativeEmail",
+    )
+    verified_secondary_email: Optional[List[str]] = Field(
+        default=None,
+        description="Sourced from the user's SecondaryAuthoritativeEmail",
+    )
+    vnet: Optional[str] = Field(
+        default=None,
+        description='VNET specifier information',
+    )
+    xms_pdl: Optional[str] = Field(
+        default=None,
+        description='Preferred data location',
+    )
+    xms_pl: Optional[str] = Field(
+        default=None,
+        description='User-preferred language',
+    )
+    xms_tpl: Optional[str] = Field(
+        default=None,
+        description='Tenant-preferred language',
+    )
+    ztdid: Optional[str] = Field(
+        default=None,
+        description='Zero-touch Deployment ID',
+    )
+
+    # V1.0 only
+    acr: Optional[Literal['0', '1']] = Field(
+        default=None,
+        description='A value of 0 for the "Authentication context class" claim indicates the end-user authentication '
+        'did not meet the requirements of ISO/IEC 29115. Only available in V1.0 tokens',
+    )
+    # V1.0 only
+    amr: Optional[List[str]] = Field(
+        default=None,
+        description='Identifies the authentication method of the subject of the token. Only available in V1.0 tokens',
+    )
+    # V1.0 only
+    appid: Optional[str] = Field(
+        default=None,
+        description='The application ID of the client using the token. Only available in V1.0 tokens',
+    )
+    # V1.0 only
+    appidacr: Optional[Literal['0', '1', '2']] = Field(
+        default=None,
+        description='Indicates authentication method of the client. Only available in V1.0 tokens',
+    )
+    # V1.0 only
+    unique_name: Optional[str] = Field(
+        default=None,
+        description='Provides a human readable value that identifies the subject of the token. Only available in V1.0 tokens',
+    )
+
+    # V2.0 only
+    azp: Optional[str] = Field(
+        default=None,
+        description='The application ID of the client using the token. Only available in V2.0 tokens',
+    )
+    # V2.0 only
+    azpacr: Optional[Literal['0', '1', '2']] = Field(
+        default=None,
+        description='Indicates the authentication method of the client. Only available in V2.0 tokens',
+    )
+    # V2.0 only
+    preferred_username: Optional[str] = Field(
+        default=None,
+        description='The primary username that represents the user. Only available in V2.0 tokens',
+    )
+
+
+class User(Claims):
+    claims: Dict[Any, Any] = Field(
```

Comment:
Being picky, I'd changed that to `Dict[str, Any]`

Reply:
```suggestion
    claims: Dict[str, Any] = Field(
```

Reply:
Updated it.